app.js

```diff
@@ -23,13 +24,14 @@ app.use((req, res, next) => {
 })
 
 app.use(cors())
-app.use(express.json())
+app.use(express.json({ limit: '5mb' }));
```

Comment:
Please make this param dynamic (read from .env file) since we are not sure if there could be any need later (I think it's likely so) to submit files greater than 5MB.

Reply:
In fact, please do not impose such a rule globally. Please add some checking on the email view alone (and it should be on frontend). It is likely people have to submit large media files such as videos to the media editor